core/ecschema-metadata/src/UnitConversion/UnitConversion.ts

```diff
@@ -0,0 +1,93 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Constant, Unit } from "../ecschema-metadata";
+import { SchemaItemType } from "../ECObjects";
+import { Float } from "./Float";
+
+/**
+ * Class used for building unit conversions, storing how to get from source unit to target unit, and evaluating input and converting it to output
```

Comment:
Think about how a user will use this class.  The documentation should focus on that.  They will use it to store the calculated conversion between two units then convert values from one to the other.  maybe point them to the API that generates the UnitConversion.

They don't need to know about the internals of the class